transports/jms/src/main/java/org/mule/transport/jms/JmsConnector.java

```diff
@@ -1635,4 +1641,9 @@ public void setStillConnectingReceivers(boolean aBoolean)
     {
         this.stillConnectingReceivers.set(aBoolean);
     }
+
+    public boolean mustRecycleReceivers()
+    {
+        return false;
```

Comment:
isn't this changing the default behavior?

Reply:
Yes. It should be true.